### DISCUSSION TITLE
Design a beautiful side menu for a multi-vendor or single-vendor app

### DISCUSSION THREAD

**Discussion by Thedeep1** [inspireui/support#5321] [Working on Features]
You should add a beautiful side list and not the one in the multi-vendor application, it is very ugly and very old, please update it in a new and beautiful way, thank you

**Comment by Thedeep1** [comment]
important feature

**Comment by minhcasi** [comment]
Please check the medium blog on v2.5.3 - https://medium.com/inspireui/announcing-fluxstore-2-5-x-83e13a8be843

**Comment by ecomdeveloperbd** [comment]
The Left Menu is more flexible to customize:
Zoom Drawer with background:

code:

"Drawer": {
  "textColor": "FFC62828",
  "iconColor": "FFC62828",
  "zoomConfig":{
    "style": "style1",
    "backgroundImage": "https://www.desktopbackground.org/download/360x640/2012/10/01/460985_orange-yellow-blue-material-design-wallpapers-h_1600x900_h.jpg",
    "enableShadow": true,
    "angle": -12
  },

Screenshot:
https://i.imgur.com/4afmSB9.jpg
https://i.imgur.com/xMebBEZ.png
